src/analysis/trampolines.rs

```diff
@@ -147,6 +152,10 @@ pub fn analyze(
         inhibit,
         concurrency,
         is_notify,
+        bound_name: 'A', // TODO: remove that
```

Comment:
Why is it needed in addition to the normal bounds character generation we have?

Reply:
It is needed to know which bound has been assigned to the current trampoline. It's used in `codegen` when we generate C function wrappers. I'll add a comment to state this.